epilepsy12/common_view_functions/calculate_kpi_functions/score_kpi_3.py

```diff
@@ -15,9 +15,14 @@ def score_kpi_3(registration_instance, age_at_first_paediatric_assessment) -> in
 
     Calculation Method
 
-    Numerator = Number of children ([less than 3 years old at first assessment] AND [diagnosed with epilepsy] OR (number of children and young people diagnosed with epilepsy who had [3 or more maintenance AEDS] at first year) OR (Number of children less than 4 years old at first assessment with epilepsy AND myoclonic seizures)  OR (number of children and young people diagnosed with epilepsy  who met [CESS criteria] ) AND had [evidence of involvement of a paediatric neurologist] OR [evidence of referral or involvement of CESS]
-
-    Denominator = Number of children [less than 3 years old at first assessment] AND [diagnosed with epilepsy] OR (number of children and young people diagnosed with epilepsy who had [3 or more maintenance AEDS] at first year )OR (number of children and young people diagnosed with epilepsy  who met [CESS criteria] OR (Number of children less than 4 years old at first assessment with epilepsy AND  [myoclonic seizures])
+    Numerator = Number of children (
+                    [less than 3 years old at first assessment] AND [diagnosed with epilepsy]
+                     OR (number of children and young people diagnosed with epilepsy who had [3 or more maintenance AEDS] at first year) OR
+                     (Number of children less than 4 years old at first assessment with epilepsy AND (has generalised myoclonic seizures OR has focal myoclonic seizures))  OR
+                     (number of children and young people diagnosed with epilepsy  who met [CESS criteria] ) AND had
+                     [evidence of involvement of a paediatric neurologist] OR [evidence of referral or involvement of CESS]
+
+    Denominator = Number of children [less than 3 years old at first assessment] AND [diagnosed with epilepsy] OR (number of children and young people diagnosed with epilepsy who had [3 or more maintenance AEDS] at first year )OR (number of children and young people diagnosed with epilepsy  who met [CESS criteria] OR (Number of children less than 4 years old at first assessment with epilepsy AND  (has generalised myoclonic seizures OR has focal myoclonic seizures))
```

Comment:
Probably not something we need to change for this PR but I wonder if perhaps we could only document Calculation Method in one place, either in the codebase/docstrings OR in the MkDocs doco? If there is a need to be able to easily refer from one to the other we can put permalinks in the 'other' one? views @eatyourpeas @mbarton? Is this just my OCD speaking?